lib/Doctrine/DBAL/Driver/PDOSqlsrv/Statement.php

```diff
@@ -17,11 +17,20 @@ class Statement extends PDO\Statement
      */
     public function bindParam($param, &$variable, $type = ParameterType::STRING, $length = null, $driverOptions = null)
     {
-        if (
-            ($type === ParameterType::LARGE_OBJECT || $type === ParameterType::BINARY)
-            && $driverOptions === null
-        ) {
-            $driverOptions = \PDO::SQLSRV_ENCODING_BINARY;
+        switch ($type) {
+            case ParameterType::LARGE_OBJECT:
+            case ParameterType::BINARY:
+                if ($driverOptions === null) {
+                    $driverOptions = \PDO::SQLSRV_ENCODING_BINARY;
+                }
+
+                break;
+
+            case ParameterType::ASCII:
+                $type          = ParameterType::STRING;
```

Comment:
I changed `PDO::PARAM_STR` to `ParameterType::STRING` during the rebase since the direct usage of `PDO` binding types in calls to DBAL is deprecated (although may work since they use the same underlying vaules).

Reply:
Makes sense 👍